cypress/support/constants/constants.js

```diff
@@ -303,6 +303,7 @@ module.exports = {
   PLATFORM_NOT_SUPPORT_LOG: 'Platform does not support method',
   PLATFORM_NOT_TRIGGER_EVENT: 'Platform MUST not trigger event ',
   PLATFORM_TRIGGER_EVENT: 'Platform MUST trigger event ',
+  PLAYERID: 'playerId',
   PREREQUISITE_DATA: 'PreRequisiteData.json',
```

Comment:
not sure why we keep adding player in FCS. This has to be removed

Reply:
removed